chat-widget/src/components/footerstateful/downloadtranscriptstateful/interfaces/IWebChatTranscriptConfig.ts

```diff
@@ -0,0 +1,29 @@
+export interface IWebChatTranscriptConfig {
+    disabled?: boolean;
+    /**

```

Comment:
nit: Can we add line breaks in between the previous props and the next prop's comments? For better separation / ease of readability.